.gitattributes

```diff
@@ -1 +1,3 @@
 CHANGELOG.md merge=union
+* text=auto
```

Comment:
Based on my reading of the docs, this will only work if the files are not committed  CRLF.  

>When text is set to "auto", the path is marked for automatic end-of-line conversion. If Git decides that the content is text, its line endings are converted to LF on checkin. When the file has been committed with CRLF, **no conversion is done**.
```

Reply:
From https://www.git-scm.com/docs/gitattributes:

```none
If you want to ensure that text files that any contributor introduces to the repository have their line endings normalized, you can set the text attribute to "auto" for all files.

*	text=auto
```

The actual conversion of files that have CRLF (or worse mixed EOL) will be part of https://github.com/PowerShell/PowerShell/issues/4910 which we'll do after 6.0.0 final since it touches so many files.